Classes/book.rb

```diff
@@ -0,0 +1,15 @@
+require './item'
+
+class Book < Item
+  attr_accessor :publisher, :cover_state
+

```

Comment:
With attr_accessor we can read and write property value. I think all properties of the Book class should be set up in the constructor method. I suggest you change `attr_accessor :publisher, :cover_state` to `attr_reader :publisher, :cover_state`.

Reply:
@developertariq thank you for the suggestion, but I have already defined all the properties in the constructor. Can you please recheck? Regards.